package.json

```diff
@@ -94,12 +94,15 @@
     "husky": "^4.2.5",
     "jest": "^24.5.0",
     "lint-staged": "^10.1.3",
+    "ng-packagr": "^9.1.0",
+    "npm-run-all": "^4.1.5",
     "prettier": "^2.0.4",
     "rimraf": "^2.6.2",
     "rxjs": "~6.5.5",
     "serve": "^11.3.0",
     "start-server-and-test": "^1.10.11",
     "ts-jest": "^24.0.0",
+    "tsickle": "^0.38.1",
```

Comment:
Why is this needed?

Reply:
it's a part of ng-packagr, it adds jsdoc annotations for the closure compiler.

Reply:
If it's part of ng-packagr, then why do we need to install it separately from ng-packagr?

Reply:
Because it's not a peer dependency of ng-packagr.